.gitignore

```diff
@@ -15,6 +15,8 @@
 *.egg-info
 *.pyc
 *.so
+*.lib
+*.dll
```

Comment:
This change seems unrelated to the PR

Reply:
I could remove it. 
Since I wrote the code under windows, without this, it will just be a little bit painful to make a commit.

Reply:
Yes kindly remove this change and rebase on top of `develop`, I added these files in f8cb3a352015c24b05de01a228d2cde54ae71a64